xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -581,11 +581,13 @@ void CMusicInfoTag::SetArtist(const CArtist& artist)
 {
   SetArtist(artist.strArtist);
   SetAlbumArtist(artist.strArtist);
+  SetMusicBrainzArtistID({ artist.strMusicBrainzArtistID });
   SetGenre(artist.genre);
-  m_dateAdded = artist.dateAdded;
-  m_iDbId = artist.idArtist;
-  m_type = MediaTypeArtist;
-  m_bLoaded = true;
+  SetMood(StringUtils::Join(artist.moods, " / "));
```

Comment:
You're right, I'm too much used to do this for JSON-RPC where we always want to use the same separator. `CAdvancedSettings::m_musicItemSeparator` and `CAdvancedSettings::m_videoItemSeparator` should theoretically only be used for concatenations that are shown in the GUI where it can't hurt anything. But you know our code base... ;-)